Dockerfile

```diff
@@ -26,6 +26,8 @@ RUN \
     pip3 install . --no-deps
 
 RUN mkdir -p /usr/share/freshmaker && cp contrib/freshmaker.wsgi /usr/share/freshmaker/
+# Delete the default logging configuration
+RUN rm -f fedmsg.d/freshmaker-logging.py
```

Comment:
It might be better not use `-f` flag here which would indicate error in case the current directory is different or the file no longer exists in the repo.